pkg/apis/core/types.go

```diff
@@ -422,16 +422,15 @@ type PersistentVolumeClaimList struct {
 // PersistentVolumeClaimSpec describes the common attributes of storage devices
 // and allows a Source for provider-specific attributes
 type PersistentVolumeClaimSpec struct {
+	// Resources represents the minimum resources required
+	Resources ResourceRequirements
 	// Contains the types of access modes required
```

Comment:
we need to change v1 types in staging folder and regenerate docs and other files (usually `make update`)

Reply:
Done. I dont understand why so many files being updated....
Can you help me to understand the relationship between staging and pkg/apis?

Reply:
The pkg/core types are internal representation of these objects. Most validations and admission hooks (basically api-server components) work with these types. v1 (stuff inside staging) is basically representation of these objects for clients to consume.  That is how any external component that wants to use these types should consume them.